src/main/cluster.ts

```diff
@@ -76,6 +76,7 @@ export class Cluster implements ClusterModel, ClusterState {
    * If extension sets this it needs to also mark cluster as enabled on activate (or when added to a store)
    */
   public ownerRef: string;
+  public isDead = false;
```

Comment:
I still don't think that a cluster should be created at all if the kube config is invalid or non-existant.